tools/docker/sanity_check.Dockerfile

```diff
@@ -81,11 +81,10 @@ RUN touch /ok.txt
 # docs tests
 FROM python:3.6 as docs_tests
 
-RUN pip install tensorflow-cpu==2.1.0
-RUN pip install typeguard==2.7.1
+RUN pip install tensorflow-cpu==2.1.0 typeguard==2.7.1
```

Comment:
Same here, tensorflow is ~100MB, if we change the typeguard version, we don't want to download tensorflow again.